### ISSUE TITLE
Initial blank screen?

### ISSUE THREAD

**Issue by shrinathwy** [rygine/looper#8]
The first image loads only after the js loads and starts kicking. The screen looks pretty blank till then. Is there a way to fix it?

**Comment by fedegonzaleznavarro** [comment]
I was about to write a new issue but I saw this and might have to do with the same thing.
If I start  looper.js paused [ data-looper="pause" ] Nothing shows up, then when I click on the arrows it seems to start.
I'm using [ data-interval="300000"] for now, but it would be nice to be able to start it paused and have it show up.
Thanks for sharing ! :)

**Comment by cmc-tech** [comment]
I get a blank screen, all the time, when I follow the directions for install. Checked to make sure would display images without script and CSS, and that worked. As soon as I link the CSS and JavaScript, it's dead! Please see the the jsBin <a href="http://jsbin.com/niweqozo/2/">http://jsbin.com/niweqozo/2/</a> if you need more info. Thanks.

**Comment by rygine** [comment]
@cmc-tech You need to specify a width/height for your looper container. In your jsbin example, try setting the width/height on the .looper element.

@fedegonzaleznavarro You can disable the interval with data-interval="false"

@shrinathwy The images are hidden by default with the CSS. Perhaps you could add a custom loader until the JS loads?
